src/xdocs/checks/annotation/index.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+
+  <head>
+    <title>Annotations</title>
+  </head>
+
+  <body>
+    <section name="Annotation Checks">
+      <div class="wrapper">
+        <table>
+          <tr>
+            <td>
+              <a href="checks/annotation/annotationlocation.html#AnnotationLocation">
```

Comment:
I have to change to absolute URLs `/checks/annotation/annotationlocation.html#AnnotationLocation`

Reply:
Done.